model/riscv_insts_zvbb.sail

```diff
@@ -404,7 +406,8 @@ mapping clause assembly = VROR_VV(vm, vs1, vs2, vd)
  <-> "vror.vv" ^ spc() ^ vreg_name(vd) ^ sep() ^ vreg_name(vs2) ^ sep() ^ vreg_name(vs1) ^ maybe_vmask(vm)
 
 function clause execute (VROR_VV(vm, vs1, vs2, vd)) = {
-  let SEW      = get_sew();
+  let SEW_pow  = get_sew_pow();
```

Comment:
Unfortunate that we have to make this change but IMO the other cleanups are worth it.